src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+  </head>
+  <body>
+    `Hello, world!` 
```

Comment:
```suggestion
    Hello, world!
```
There is no need to use quotes inside the tag.

Reply:
Chenges added)

Reply:
You mean like this
![Screenshot_2](https://user-images.githubusercontent.com/107686556/178295523-f40f1bee-e98e-45e1-a62f-3cff4006815c.png)
?